### PR TITLE
Update to Alpine Linux 3.19 and improved README to show size differences for alpine versions.

### PR DESCRIPTION
**Pull request by parasew**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Start with a rust alpine image
-FROM rust:1-alpine3.16
+FROM rust:1-alpine3.15
```
Comment (kpcyrd) [inline Dockerfile]:
```suggestion
FROM rust:1-alpine3.19
```

**Comment by kpcyrd** [comment]
Thanks! I'm not sure if historic data is relevant though, using end-of-life Alpine/Rust releases is not adviced (even if they might be slightly smaller).

**Comment by parasew** [comment]
I changed the Alpine version to 3.19, as you suggested. I also added your comment about not using older (eol) Alpine versions, even if they are smaller in size. The historic table is relevant imho, as it shows how Alpine docker image sizes have changed over the course of the last versions. I see the whole repo as an educational resource (so it can help people to learn about Docker, Alpine and Rust). Hope this suits.

**Comment by kpcyrd** [comment]
Thanks!

**Pull request merged**